packages/wrangler/scripts/generate-json-schema.ts

```diff
@@ -4,8 +4,8 @@ import { createGenerator } from "ts-json-schema-generator";
 import type { Config, Schema } from "ts-json-schema-generator";
 
 const config: Config = {
-	path: join(__dirname, "../src/config/config.ts"),
-	tsconfig: join(__dirname, "../tsconfig.json"),
+	path: join(__dirname, "../../workers-utils/src/config/config.ts"),
+	tsconfig: join(__dirname, "../../workers-utils/tsconfig.json"),
```

Comment:
Should these be using `createRequire` to get access to the absolute path?
How are these files used at runtime?